docs/platforms/unity/data-management/data-collected.mdx

```diff
@@ -0,0 +1,76 @@
+---
+title: Data Collected
+description: "See what data is collected by the Sentry SDK."
+sidebar_order: 1
+---
+
+Sentry takes data privacy very seriously and has default settings in place that prioritize data safety, especially when it comes to personally identifiable information (PII) data. When you add the Sentry SDK to your application, you allow it to collect data and send it to Sentry during the runtime of your application.
+
+The category types and amount of data collected vary, depending on the integrations you've enabled in the Sentry SDK. Here's a list of data categories the Sentry .NET SDK collects:
+
+## HTTP Headers
+
+By default, the Sentry SDK sends HTTP headers to Sentry but redacts any headers that may contain sensitive data. (See the [list of headers](https://github.com/getsentry/sentry-dotnet/blob/main/src/Sentry/Internal/RedactedHeaders.cs#L7) that are redacted).
+
+## Cookies
+
+By default, the Sentry SDK doesn't send cookies.
+
+If you want to send cookies, set the option `SendDefaultPii = true` when initializing the Sentry SDK.
+
+A [data scrubber](/platforms/dotnet/data-management/sensitive-data/) can be used to filter information before sending data to Sentry.
+
+## Information About Logged-in User
+
+By default, the Sentry SDK doesn't send any information about the logged-in user, such as email address, user ID, or username. Even if enabled, the type of logged-in user information you'll be able to send depends on the integrations you enable in Sentry's SDK. Most integrations won't send any user information. Some will only set the user ID, but there are a few that will set the user ID, username, and email address.
+
+To start sending logged-in user information, set the option `SendDefaultPii = true` when initializing the Sentry SDK.
```

Comment:
Not sure Unity adds User if we flip this flag. @bitsandfoxes ?

Reply:
The Unity SDK guards and sets the userID [here](https://github.com/getsentry/sentry-unity/blob/21c9d563f2d75e7cbe64061321d0fa65442b30d3/src/Sentry.Unity/Integrations/UnityScopeIntegration.cs#L172-L181).
On Desktop and WebGL, the `options.defaultUserId` resolves to Unity's [AnalyticsSessionInfo.userId](https://docs.unity3d.com/2018.3/Documentation/ScriptReference/Analytics.AnalyticsSessionInfo-userId.html). 

On mobile the SDK tries to fetch the `Installation ID` first and if it fails, it falls back to AnalyticsSessionInfo.userId as well.

Reply:
It does not. The one place the Unity SDK uses `SendDefaultPii` for guarding and setting the `DeviceUniqueIdentifier` [here](https://github.com/getsentry/sentry-unity/blob/21c9d563f2d75e7cbe64061321d0fa65442b30d3/src/Sentry.Unity/Integrations/UnityScopeIntegration.cs#L92-L94). The rest is in control of the .NET SDK.